.ci/make.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# parameters are available to this script
+
+# common build entry script for all elasticsearch clients
+
+# ./.ci/make.sh assemble VERSION
+
+script_path=$(dirname "$(realpath -s "$0")")
+repo=$(realpath "$script_path/../")
+
+# shellcheck disable=SC1090
+CMD=$1
+TASK=$1
+VERSION=$2
+STACK_VERSION=$VERSION
+set -euo pipefail
+
+output_folder=".ci/output"
+OUTPUT_DIR="$repo/${output_folder}"
+mkdir -p "$OUTPUT_DIR"
+
+RUST_TOOLCHAIN=${RUST_TOOLCHAIN-latest}
+
+echo -e "\033[34;1mINFO:\033[0m VERSION ${STACK_VERSION}\033[0m"
+echo -e "\033[34;1mINFO:\033[0m OUTPUT_DIR ${OUTPUT_DIR}\033[0m"
+echo -e "\033[34;1mINFO:\033[0m RUST_TOOLCHAIN ${RUST_TOOLCHAIN}\033[0m"
+
+echo -e "\033[1m>>>>> Build [elastic/elasticsearch-rs container] >>>>>>>>>>>>>>>>>>>>>>>>>>>>>\033[0m"
+
+docker build --build-arg RUST_TOOLCHAIN="${RUST_TOOLCHAIN}" --file .ci/DockerFile --tag elastic/elasticsearch-rs .
+
+echo -e "\033[1m>>>>> Run [elastic/elasticsearch-rs container] >>>>>>>>>>>>>>>>>>>>>>>>>>>>>\033[0m"
+
+case $CMD in
+    assemble)
+        TASK=release
+        ;;
+    *)
+        echo -e "\nUsage:\n\t $CMD is not supported right now\n"
+        exit 1
+esac
+
+docker run \
+  --env "CI=true" \
+  --name test-runner \
+  --volume "${OUTPUT_DIR}:/usr/src/elasticsearch-rs/${output_folder}" \
+  --volume ${repo}/test_results:/usr/src/elasticsearch-rs/test_results \
+  --rm \
+  elastic/elasticsearch-rs \
+  echo 'something calling automation here using $TASK and $VERSION producing output in $OUTPUT_DIR'
```

Comment:
what should the output be? The packaged crates that can be published at a later point? If so, the command is 

```
cargo make package --env "CARGO_MAKE_CARGO_PACKAGE_FLAGS=--target-dir ${OUTPUT_DIR}"
```

This will package a version specified in the Cargo.toml of elasticsearch crate, which is incremented manually, at present. What is the proposed flow?

Reply:
This will be called by unified build to stage artifacts so some form of artifact validation might be good too (e.g version and metadata is as expected). Version bumps are assumed to happen after FF for majors + minors or after release for patches ahead of time.

The unified release will also git tag + github release when the release goes out. We are going to be using github actions to automate the actual staging of the artifact and attaching release notes.